package.json

```diff
@@ -15,6 +15,7 @@
         "@types/react": "^17.0.34",
         "@types/react-dom": "^17.0.11",
         "@types/react-router-dom": "^5.3.2",
+        "cypress-real-events": "^1.7.1",
```

Comment:
I think that this library is not needed anymore, so it's best to just delete it via `npm uninstall cypress-real-events` command. Also you should delete all references to this library from previous files.